server/config/environment/production.js

```diff
@@ -13,6 +13,13 @@ module.exports = {
             process.env.PORT ||
             8080,
 
+  // Server Address
+  addr:     process.env.SERVER_ADDR ||
+            "",
+
+  // Sendgrid email
+  sendgridApiKey: process.env.SENDGRID_API_KEY,
```

Comment:
this doesn't have to be in the index and the other environments? Why not only have this code in environment/index.js?